docs/content/en/docs/install/install.md

```diff
@@ -109,10 +109,9 @@ For more information,see
 
 ## Use manifests
 
-All versions of the Lifecycle Toolkit can be installed using manifests,
+Versions v0.10.0 and earlier of the Lifecycle Toolkit can be installed using manifests,
 although we recommend that you use Helm Charts
-to install Version 0.7.0 and later
-because the Helm Charts allow you to customize your configuration.
+because they allow you to easily customize your configuration.
```

Comment:
I commented about this block for https://github.com/keptn/lifecycle-toolkit/pull/1850 -- not sure why it is two PRs...

Reply:
we have charts from klt 0.7.0 so it should be fine